WordPress/Classes/ViewRelated/Blog/Blogging Reminders/BloggingRemindersAnimator.swift

```diff
@@ -0,0 +1,37 @@
+/// A transition animator that moves in the pushed view controller horizontally.
+/// Does not handle the pop animation since the BloggingReminders setup flow does not allow to navigate back.
+class BloggingRemindersAnimator: NSObject, UIViewControllerAnimatedTransitioning {
+
+    func transitionDuration(using transitionContext: UIViewControllerContextTransitioning?) -> TimeInterval {
+        return 0.2
+    }
+
+    func animateTransition(using transitionContext: UIViewControllerContextTransitioning) {
+
+        guard let sourceViewController =
+                transitionContext
+                .viewController(forKey: UITransitionContextViewControllerKey.from),
```

Comment:
Nitpicky: The line breaking between `transitionContext` and its method `.viewController(...)` seems a bit nonstandard.  IMHO it makes readability suffer slightly.

Any thoughts on standardizing this to be:

```swift
guard let sourceViewController =
        transitionContext.viewController(forKey: UITransitionContextViewControllerKey.from)
```

Reply:
sure